frontend/src/languages/english.js

```diff
@@ -6,7 +6,7 @@ const en = {
       thousand: ",",
       precision: 2
     },
-    dateFormat: "dd/MM/yyyy",
+    dateFormat: "DD/MM/YYYY",
```

Comment:
Date format in Englisch per default should be MM/DD/YYYY

Reply:
nope, we use british english, so the format is correct:

https://en.wikipedia.org/wiki/Date_and_time_notation_in_the_United_Kingdom